repos/system_upgrade/el7toel8/actors/checkmemory/tests/test_checkmemory.py

```diff
@@ -0,0 +1,59 @@
+import os
+from collections import namedtuple
+
+import pytest
+
+from leapp import reporting
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.actor import library
+from leapp.libraries.common import testutils
+from leapp.libraries.common.config import architecture
+from leapp.libraries.stdlib import api
+from leapp.models import MemoryInfo
+
+
+class CurrentActorMocked(object):
+    def __init__(self, arch):
+        self.configuration = namedtuple('configuration', ['architecture'])(arch)
+
+    def __call__(self):
+        return self
+
+
+def mocked_get_memoryinfo(filename):
+    """
+    Return lines of the self.filename test file located in the files directory.
+
+    Those files contain /proc/meminfo content from machine.
+    """
+    memory_info = MemoryInfo()
+    mem_info = dict((i.split()[0].rstrip(':'), int(i.split()[1])) for i in open(os.path.join('tests/files', filename), 'rb').readlines())  # noqa: E501; pylint: disable=line-too-long
```

Comment:
You should have a very good reason to skip linter checks. Why disable=line-too-long should be chosen over code refactor?

Reply:
fixed.